cmd/bcpd/app/benchmark.txt

```diff
@@ -0,0 +1,26 @@
+BenchmarkSendTx/100-10-8                     300           5112629 ns/op         1199393 B/op      24288 allocs/op
+BenchmarkSendTx/100-100-8                     50          34253808 ns/op         2543190 B/op      47555 allocs/op
+BenchmarkSendTx/10000-10-8                   200          14787621 ns/op         3740037 B/op      76124 allocs/op
+BenchmarkSendTx/10000-100-8                   30          48618981 ns/op         6845716 B/op     124288 allocs/op
+BenchmarkSendTx/10000-1000-8                   3         404855149 ns/op        39829581 B/op     653600 allocs/op
+BenchmarkSendTx/100000-10-8                  100          77092694 ns/op        17602598 B/op     355118 allocs/opBenchmarkSendTx/100000-100-8                  30          87291256 ns/op        16496994 B/op     290574 allocs/op
+BenchmarkSendTx/100000-1000-8                  3         498527483 ns/op        77594389 B/op    1062805 allocs/opPASS
+ok      github.com/iov-one/weave/cmd/bcpd/app   106.689s
+
+------------------------------------------------------------
+
+BenchmarkSendTxMultiSig/10000-100-100-2-1-8                   50          37669548 ns/op         3114398 B/op      61234 allocs/op
```

Comment:
Numbers here seem more reasonable. With more contracts in storage adding a small (eg. 20-30%) overhead to the time, nothing like 10x

Same as above, I am doing mental math with decimals to covert block time to average tx time in order to compare block sizes of 100 and 1000